src/clr-angular/timeline/timeline-layout.enum.ts

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+export enum TimelineLayout {
```

Comment:
This is used as an input for ClrTimeline, it needs to be exported so consumers can use it.